docs/machine-learning/automate-training-with-cli.md

```diff
@@ -31,7 +31,10 @@ Currently, the ML Tasks supported by the ML.NET CLI are:
 - `binary-classification`
 - `multiclass-classification`
 - `regression`
-- Future: other machine learning tasks such as `recommendation`, `ranking`, `anomaly-detection`, `clustering`
+- `recommendation`
+- `ranking`
+- `anomaly-detection`
+- `clustering`
```

Comment:
```suggestion
- `recommendation`
```

Reply:
Ah thanks for catching that @luisquintanilla. I missed that this was only for AutoML. I believe you're correct.